src/material/input/input.ts

```diff
@@ -232,7 +234,8 @@ export class MatInput extends _MatInputMixinBase implements MatFormFieldControl<
     _defaultErrorStateMatcher: ErrorStateMatcher,
     @Optional() @Self() @Inject(MAT_INPUT_VALUE_ACCESSOR) inputValueAccessor: any,
     private _autofillMonitor: AutofillMonitor,
-    ngZone: NgZone) {
+    ngZone: NgZone,
+    @Attribute('aria-describedby') private _initialAriaDescribedBy?: string|null) {
```

Comment:
@mmalerba This is more of a proposal at this point. I feel slightly unhappy that we'd need to do this for all custom form-field controls. I think that it would be correct though to not override static `aria-described` values.

Possible solutions:

1. Go with the one currently outlined in this PR. That might be cumbersome/repetitive to add to all custom controls. There is some duplication anyway already.. and it's inevitable to elide all of it.
2. Pass the initial described by value to the abstract form-control, and then the form-field will merge it automatically. That means less work for custom controls (except for still doing the attribute injection)
3. Accept `aria-described` on the form-field and merge it there (before calling `setDescribedByIds`). That is most convenient for controls, but less for users as we'd still silently override existing `aria-describedby` values.

Note that all of these approaches have the downside of not supporting dynamically changing `aria-describedby` values. It wouldn't be easy to detect these attribute changes, unless we make it an input. I have a feeling that the ideal solution would be **(2)** with the use of an `@Input('aria-describedby`) userAriaDescribedBy: string = ''`. That would be the only line custom controls would need to add then.

Reply:
I would probably add an `@Input` to the `matInput`, you _can_ use it outside of the form field, but that's not something we go out of our way to support

Reply:
yeah, that's fair. We can make it clear with a small comment on the input. I'll experiment with it. thx for the feedback so far.

Reply:
@mmalerba Updated the proposal. Please have another look when you get a chance.